### PR TITLE
Document changes to integration test setup in Contrib Guide

### PR DESCRIPTION
**Pull request by fhennig**

### CODE REVIEW

File: modules/contributor/pages/steps.adoc [hunk 1]
```diff
@@ -52,6 +52,8 @@ kind load docker-image docker.stackable.tech/stackable/<operator>:<version>-nigh
 helm install <operator> deploy/helm/<operator>/
 ----
 
+5. The tests are run with the `scripts/run_tests.sh` script.
```
Comment (maltesander) [inline modules/contributor/pages/steps.adoc:55]: I would mention that this has to be executed from the repository root folder like ./scripts/run_tests.sh

**Review by maltesander — APPROVED** [review]
LGTM.

File: modules/contributor/pages/steps.adoc [hunk 2]
```diff
@@ -13,7 +13,7 @@ Depending on the project, not all steps make sense. For instance, if the documen
 == Changes in Rust projects
 
 1. Make your desired changes in the according repository and test them manually. Ensure that the code compiles without warnings (`cargo clippy --all-targets`) and that the code is formatted with `cargo fmt`.
-2. If code was added or adapted then please create or adapt the unit tests in the same file. Ensure that all unit tests run successfully (`cargo test`).
+2. If code was added or adapted then please create or adapt the unit tests in the same file as well as the integration tests in the `tests` directory. Ensure that all unit tests run successfully (`cargo test`) and all integration tests run successfully (`./scripts/run_tests.sh`). See also <<_changes_in_the_integration_tests>>.
```
Comment (razvan) [inline modules/contributor/pages/steps.adoc:16]: No "please", please :)
```suggestion
2. If you made source code changes, also update the unit tests in the same file as well as the integration tests in the `tests` directory. Ensure that all unit tests run successfully (`cargo test`) and all integration tests run successfully (`./scripts/run_tests.sh`). See also <<_changes_in_the_integration_tests>>.
```

**Review by razvan — COMMENTED** [review]
One suggestion

**Pull request merged**